internal/service/alarms/internal/dictionary/dictionary.go

```diff
@@ -0,0 +1,190 @@
+package dictionary
+
+import (
+	"context"
+	"fmt"
+	"log/slog"
+	"sync"
+
+	monitoringv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/selection"
+	clusterv1 "open-cluster-management.io/api/cluster/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/openshift-kni/oran-o2ims/internal/clients"
+)
+
+const (
+	managedClusterVersionLabel = "openshiftVersion-major-minor"
+	localClusterLabel          = "local-cluster"
+
+	resourceTypeCluster = "Cluster"
```

Comment:
- nit: maybe a separate block since it's unrelated to acm values (this is o-ran specific stuff)? Also maybe a comment about what this is how we will have more types in the future?